### ISSUE TITLE
⌨ Secondary `ToggleGroup` has insufficient colour contrast

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#6763]
## ⌨ Accessibility Issue

Our `ToggleGroup` component has insufficient colour contrast when the colour is set to `secondary` 

### WCAG 2.1 Criteria

[Elements must have sufficient color contrast](https://dequeuniversity.com/rules/axe/4.7/color-contrast?application=axeAPI)

## 🦋 Expected Behaviour

The secondary `ToggleGroup` should either have

1. Darker background
2. Darker text (black?)

## 📋 Steps to Reproduce

1. Open the "Toggle group single" story
3. Confirm the secondary colour's background is too light for white text

## 📸 Screenshot

![Screenshot 2023-05-31 093404](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/27272a99-41d9-42af-bda4-fdf35f152f71)

## 🙋‍♀️ Proposed Solution

- [ ] Work with designers to determine the new colours and implement them

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the issue was addressed.

- [ ] Secondary `ToggleGroup` has colours adjusted for a min contrast ratio of 4.5:1

**Comment by esizer** [comment]
Fixed by #6814
